src/pages/CovidHistory.js

```diff
@@ -239,56 +250,64 @@ const CovidHistory = ({
       <div className="health-checklist">
-        {covidHistory.map((history, indx) => {
-          return (
-            <>
-              {history.field !== "dateCovidBefore" && (
-                <div
-                  className={
-                    `${history.type}` === "Boolean"
-                      ? "history-list-content"
-                      : "input-history"
-                  }
-                  key={getRandomKey()}
-                >
-                  {history.type === "Text" && <label>{history.title}</label>}
-                  {history.type === "Text" ? (
-                    <div className="date-input">
-                      <input
-                        className={
-                          history.field === "dateOfDose1Vaccination"
-                            ? doseOneClass
-                            : doseTwoClass
-                        }
-                        type="date"
-                        id={indx}
-                        name={history.field}
-                        placeholder={DATE_FORMAT.mmddyyyy}
-                        onChange={handleInputChange}
-                        max={moment().format(DATE_FORMAT.yyyymmdd)}
-                        disabled={checkDisabled(history.field)}
-                        value={getValue(history.field)}
-                      />
-                    </div>
-                  ) : (
-                    <input
-                      className="symptoms-checkbox"
-                      type="checkbox"
-                      id={indx}
-                      name={history.field}
-                      value={history.field}
-                      onChange={handleCheckboxChange}
-                      checked={intakeState[history.field]}
-                    />
-                  )}
-                  {history.type === "Boolean" && (
-                    <label htmlFor={history.field}>{history.title}</label>
-                  )}
-                </div>
-              )}
-            </>
-          );
-        })}
+        <>
+          <div className="history-list-content">
+            <input
+              className="symptoms-checkbox"
+              type="checkbox"
+              name="covidVaccinationDose1Taken"
+              value="covidVaccinationDose1Taken"
+              onChange={handleCheckboxChange}
+              checked={intakeState["covidVaccinationDose1Taken"]}
+            />
+            <label htmlFor={"covidVaccinationDose1Taken"}>
+              Vaccination - 1st dose
+            </label>
+          </div>
+          <div className="input-history">
+            <label>Date of dose - 1st vaccination</label>
+            <div className="date-input">
+              <input
+                className={doseOneClass}
+                type="date"
+                name={"dateOfDose1Vaccination"}
+                placeholder={DATE_FORMAT.mmddyyyy}
+                onChange={handleInputChange}
+                max={moment().format(DATE_FORMAT.yyyymmdd)}
```

Comment:
please move moment().format(DATE_FORMAT.yyyymmdd) to a variable and reuse everywhere for dates